src/Features/CSharp/Portable/EditAndContinue/CSharpEditAndContinueAnalyzer.cs

```diff
@@ -1989,113 +2014,125 @@ private void ClassifyReorder(SyntaxNode newNode)
 
             private void ClassifyInsert(SyntaxNode node)
             {
+                // Check edits that are rude as top level or statement edits
                 switch (node.Kind())
                 {
-                    case SyntaxKind.GlobalStatement:
-                        // TODO:
+                    // These are rude when added to methods, or local functions
+                    case SyntaxKind.TypeParameter:
```

Comment:
>case SyntaxKind.TypeParameter: [](start = 20, length = 30)

Why is SyntaxKind.Parameter not here?

Reply:
Existing tests allow inserting parameters to local functions: https://github.com/dotnet/roslyn/blob/master/src/EditorFeatures/CSharpTest/EditAndContinue/StatementEditingTests.cs#L6737-L6763